src/components/attendee/Banner/Banner.module.scss

```diff
@@ -0,0 +1,36 @@
+.banner {
+  position: fixed;
+  z-index: 90;
```

Comment:
Please replace all hard-coded `z-index` with relative functions like in `AttendeeLayout.module.scss.

```scss
$-z-index--space: layer.above(0);
$-z-index--ui: layer.above($-z-index--space);
$-z-index--popover: layer.below($-z-index--ui);
```

The reason for this is being progressively harder to determine correct order the more z-index constants are added.
This way a single one can serve as an anchor and all the rest would be relative to each other.

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2952/commits/31283fe4933ee54a28e2a97af203f53d7c7886bf